site/includes/skiplinks.hbs

```diff
@@ -7,9 +7,9 @@
 		<a class="wb-sl" href="#wb-info">{{{i18n "tmpl-skip-about-site"}}}</a>
 	</li>
 {{/isnt}}
-{{#is page.partner "true"}}
+{{#if page.secondarymenu}}
 	<li class="wb-slc visible-md visible-lg">
```

Comment:
Had to use `{{#if}}` because I am using `secondarymenu` as an object to build the local navigation.